build/dockerfiles/content_sets_centos8_appstream.repo

```diff
@@ -1,5 +1,5 @@
 [AppStream]
 name=CentOS-8 - AppStream
-baseurl=http://mirror.centos.org/centos-8/8/AppStream/x86_64/os/
+baseurl=http://mirror.centos.org/centos-8/8/AppStream/arch/os/
```

Comment:
don't you need `$arch` there?

Reply:
arch is just a placeholder declared inside content_sets_centos8_appstream.repo. Anything inside this file will act as a constant. Using $arch will not act like a normal shell variable.
It needs to be replaced by appropriate architecture (x86_64, ppc64le, etc) depending on the machine on which it is being built.
This replacement will be done by following additional instruction in the rhel.Dockerfile:
``` 
RUN ARCH=`arch` && sed -i "s|arch|${ARCH}|g" /etc/yum.repos.d/content_set*.repo
```

Reply:
yes but iirc you can embed an OS aware variable and not need to do sed later.

for example...

`baseurl=http://SERVER/content/dist/rhel8/8/$basearch/appstream/os` 

But using `arch` instead of `uname -m` is cool. Didn't know about that command. :D

Anyway please sync your efforts with the s390x team so we're not solving the same problem two ways and ending up with conflicting approaches/PRs. 

cc: @aditijadhav38